test/ACL.js

```diff
@@ -245,6 +246,11 @@ describe('Access Control Test', async () => {
     await stakeManager.revokeRole(await constants.getStakerActivityUpdaterHash(), signers[0].address);
   });
 
+  it('Only Default Admin should able to update Block Reward', async () => {
+    await assertRevert(stakeManager.connect(signers[1]).updateBlockReward(100), expectedRevertMessage);
+    assert(await stakeManager.updateBlockReward(100), "Admin not able to update BlockReward");
```

Comment:
Let's add one more case, where it `updateBlockReward` will be called by `non-admin` account & it should revert. @abhishekvispute

Reply:
Yes, that's what line no. 250 does @dev1644

Reply:
Ok, Sounds good, I overlooked it.